src/d3a/d3a_core/export.py

```diff
@@ -553,12 +556,12 @@ def _plot_supply_demand_curve(self, subdir: str, area: Area):
                                           name=time_slot.format(DATE_TIME_FORMAT +
                                                                 ' Clearing-Energy'))
                     data.append(data_obj)
-                    xmax = max(xmax, clearing_point[1]) * 3
+                    xmax = max(xmax, clearing_point[1]) * 6
```

Comment:
I think I already asked you offline, but can you remind me why do we need the hardcoded value here ?

Reply:
Ohkay, can't recall, should I move it to ConsSettings?

Reply:
Leave it as it is, not that big of a deal, I am just curious why did you need to change 3 -> 6

Reply:
I was playing around with supply/demand curve, with `3` wasn't able to fit the whole supply/demand curve wrt my setup_file `test_clearing_energy`

Reply:
Got it, thanks!